Automattic/jetpack
# Map: change dependency mapbox-gl to maplibre-gl

Closes #20127

#### Changes proposed in this Pull Request:

* Swap dependency `mapbox-gl` for `maplibre-gl`

#### Jetpack product discussion

* Internal: p7fD6U-4B8-p2

#### Does this pull request change what data or activity we track or use?

* No.

#### Testing instructions:

* On a JN site, create a new post with a Map block (this will require a Mapbox API key), briefly test out the Map block features if you are unfamiliar.
* Pull down the changes in this PR.
* Make sure the existing Map block you have works as expected, and try inserting a new one. Verify that the maps work well on the frontend as well.

**Note**: I noticed in browser console some warnings mentioning `styleimagemissing`, but these occur on latest stable as well currently, so that will be a separate issue to take a look at.
